### DISCUSSION TITLE
Picture in picture mode isn't always entered automatically

### DISCUSSION THREAD

**Discussion by alejandro-alzate** [libre-tube/LibreTube#5618] [General]
### Steps to reproduce

Using pill gesture navigation interferes with the pip mode
Sometimes the action triggers pip as in older android versions (like v8)
But sometimes the flick speed or angle puts libretube as it were in audio mode
Other times it just stops.

### Expected behavior

the expected behavior is bringing back the pip button found below the video player or stuffing it onto the video gear icon (either inside or beside it) if below the player it too crammed.
I'm geniunely curious if this is a technical issue because the new landscape mode that btw every time you rotate the screen it stops and rewinds a bit. or is an ux issue like not fitting in tiny screen or users accidentally clicking it, or is just now a disabled by deafult optional feature because the is pretty hidden since i guessed it was stuffed inside the settings, but i haven't able to find it.

### Actual behavior

When exiting with the home gesture activates pip mode inconsintenly

### LibreTube version

0.21.1

### Android version

LineageOS:13 no gms

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Removing the button was a design choice to remove unnecessary actions, the button itself is not technical issue.

For myself, I never encountered any issue with the automatic PiP activation, so removing that button was reasonable.

**Comment by alejandro-alzate** [comment]
Can i at least beg of bringing it back as an optional feature that is disabled by default? like the optional home optional buttons?

**Reply by Bnyro** [reply]
#5619

**Reply by alejandro-alzate** [reply]
I'm really grateful, and sorry for bothering.
